components/SubscriptionList/SubscriptionList.tsx

```diff
@@ -123,40 +130,50 @@ function SubscriptionList() {
         })
     }
 
+    function deleteAll() {
+        api.unsubscribeAll()
+            .then(() => {
+                setSubscriptions([])
+                toast.success('All subscriptions were sucessfully removed')
```

Comment:
subscriptions were renamed to notifications, is this called subscriptions on purpose?